### PR TITLE
LPS-154811

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

This is for ticket https://issues.liferay.com/browse/LPS-154811.
Thanks for checking

### CODE REVIEW

**Comment by jeffwu0724** [comment]
@catarineaguiar Can you help me look at the test result, please?

Thanks
Jeff

**Comment by catarineaguiar** [comment]
@jeffwu0724 no regressions found.

**Comment by tinatian** [comment]
@jeffwu0724 There is one not consistent with others:

`modules/sdk/gradle-plugins-workspace/build.gradle:	compile group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
`

File: lib/development/dependencies.properties [hunk 1]
```diff
@@ -22,7 +22,7 @@ dtdparser=com.liferay:com.wutka.dtd:1.20
 dumbster=com.liferay:com.dumbster.smtp:02856255a6cc7f
 error-prone-ant=com.google.errorprone:error_prone_ant:1.0.8
 freemarker=com.liferay:org.freemarker:2.3.29.LIFERAY-PATCHED-2
-gson=com.google.code.gson:gson:2.2.4
+gson=com.google.code.gson:gson:2.9.0
```
Comment (tinatian) [inline lib/development/dependencies.properties:25]: @jeffwu0724 There is already a shared dependency of this jar, we should remove this.

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment and resend.

Thanks.
Tina

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**